src/client/src/components/Chat.jsx

```diff
@@ -0,0 +1,75 @@
+import { TextField } from "@mui/material";
+import React, { useContext, useEffect } from "react";
+import "../css/Chat.css";
+import styled from "@mui/material";
+import SendRoundedIcon from "@mui/icons-material/SendRounded";
+import AttachFileIcon from "@mui/icons-material/AttachFile";
+
+const Chat = () => {
+  return (
+    <div className="conatiner">
+      <div className="chatbox">
+        <div className="userpic">
+          <h2 className="username">Jane Doe</h2>
+        </div>
+        <div class="vl"></div>
+        <div className="sender-message">
+          <div className="sender-userpic"></div>
+          <div className="sender-text">Hi, how are you?</div>
+        </div>
+        <div className="user-message">
+          <div className="user-text">Hi, I am doing great! What about you?</div>
+          <div className="user-userpic"></div>
+        </div>
+        <div className="message-container">
+          <input
+            className="message"
+            fullWidth
+            placeholder="Type your message here"
+          />
+          <SendRoundedIcon className="sendicon" />
+          <AttachFileIcon className="attachicon" />
+        </div>
+      </div>
+      <div className="contacts">
+        <h2 className="mycontact">My Contacts</h2>
+        <div>
+          <ul className="ul">
+            <l1 className="connectionsInfo">
+              <img
+                src="https://i.pinimg.com/originals/f1/0f/f7/f10ff70a7155e5ab666bcdd1b45b726d.jpg"
+                alt="comapnyPic"
+                className="companyPic"
+              ></img>
+              <div>
+                <span className="connectionName">John Doe</span>
+              </div>
+            </l1>
+            <l1 className="connectionsInfo">
+              <img
+                src="https://i.pinimg.com/originals/f1/0f/f7/f10ff70a7155e5ab666bcdd1b45b726d.jpg"
+                alt="comapnyPic"
+                className="companyPic"
+              ></img>
+              <div>
+                <span className="connectionName">Jane Doe</span>
+              </div>
+            </l1>
+            <l1 className="connectionsInfo">
+              <img
+                src="https://i.pinimg.com/originals/f1/0f/f7/f10ff70a7155e5ab666bcdd1b45b726d.jpg"
+                alt="comapnyPic"
+                className="companyPic"
+              ></img>
+              <div>
+                <span className="connectionName">Jack Billionaire</span>
+              </div>
+            </l1>
```

Comment:
The contacts should be centered. 
![Screen Shot 2023-03-09 at 11 17 09 AM](https://user-images.githubusercontent.com/14958097/224086038-5433c39f-caba-434a-b081-1efaaecf1e86.png)

Reply:
![image](https://user-images.githubusercontent.com/37922021/224088834-31e3d1f9-a244-41af-acd9-69789460ceb1.png)
The css is messed up on my end as well